internal/pkg/deploy/cloudformation/testdata/rendered_pipeline_cfn_template.yml

```diff
@@ -0,0 +1,378 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This is how the rendered pipeline CFN template looks like for now

Reply:
This is cool with the testdata folder!